cmd/podman/images.go

```diff
@@ -145,20 +145,20 @@ var (
 
 func imagesCmd(c *cli.Context) error {
 	var (
-		filterFuncs []image.ResultFilter
-		newImage    *image.Image
+		filterFuncs []imagefilters.ResultFilter
+		newImage    *adapter.ContainerImage
 	)
 	if err := validateFlags(c, imagesFlags); err != nil {
 		return err
 	}
 
-	runtime, err := libpodruntime.GetRuntime(c)
+	localRuntime, err := adapter.GetRuntime(c)
```

Comment:
FWIW I find it rather confusing that the `runtime` object, which may either be an actual local state, or a RPC client to a _remote_ server, is named `localRuntime` — both here, and the `runtime_remote.go:LocalRuntime` struct.

I would expect something like a `Runtime interface{…}`, with a `[Ll]ocalRuntime` and `[Rr]emoteRuntime` implementations — which could even be both compiled in at the same time, on platforms where that make sense, and allow using the same binary either for local or remote operations, depending on CLI options.